Bentheburrito/sacastats
# Codebase Cleanup pass (CharacterController)

I moved all the helper fns in CharacterControllers to new helper/context modules, `SacaStats.{Characters, Conversions, Weapons}` and optimized some of them (lmk if you have questions!). I made a bunch of other changes, like consolidating our two `maybe_to_int` fns into one under `Utils`.

### To Test
Visit all pages (especially the character ones), make sure all content looks correct
